app/models/job.ts

```diff
@@ -355,6 +356,8 @@ export class Job extends DBRecord implements JobRecord {
   collectionIds: string[];
 
   ignoreErrors: boolean;
+  
+  destination_url?: string;
```

Comment:
Is it possible to use camel case like the other properties, or was the use of underscore intentional?

Reply:
The underscore here is intentional as I was told that we have trouble with camel case column names in Postgres and want to move to field names with underscores. Chris has started to use snake case for column names in the new table `user_work`. The name of the variable actually maps to the real column name in the database table, so I used snake case here even though it is an eye sore.